README.md

```diff
@@ -322,14 +322,45 @@ You will still need to edit the blocks themselves via the sidebar.
 If you want to make the editing experience the most intuitive, you can enable real-time inplace editing, where an editor
 can change check, links or media by typing or clicking directly on your frontend instead of via fields on the sidebar.
 
-#### Inline text editing ([TODO](https://github.com/collective/volto-hydra/issues/5))
-You will add data attributes to where a blocks text is editable and where text formatting and features are locationed (like links)
-and also subscribe to ```onBlockFieldChanged``` events. This will enable handling fine grained 
-changes to text being edited such as turning text bold or creating a link. Hydra will notice where you have indicated a block field can 
-be clicked on and will automatically make it inplace editable handling typing, shortcuts 
+#### Inline text editing
+
```

Comment:
I think it needs more context because it's assuming you know about how the rich text works in volto.

> In Volto richtext is represnted by json with "nodes" that indicates what char or paragraph style has been applied and any attributes that has (such as a link for an anchor tag). It is the frontends job to turn this into the html with classes etc that you need. Instead of editing this in a non WYSIWYG way on the sidebar, you can instead make it inline editable directly in the frontend just like you could in Volto.